gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/IdentityProviderManagerImpl.java

```diff
@@ -354,7 +354,7 @@ protected boolean useJdbcRepositories() {
 
     @Override
     public boolean userProviderExists(String identityProviderId) {
-        return userProviders.containsKey(identityProviderId);
+        return userProviders.containsKey(identityProviderId) ? true : getUserProvider(identityProviderId).blockingGet() != null;
```

Comment:
Not a huge fan of this blockingGet(), maybe relying on the plugin manager will be more elegant.

Reply:
Should not possible right now due to the setInternalStatus method. Approve this change.